SCE-Development/Core-v4
# About Us Page

Created frontend/backend for About Us Page on the general website and an About Us Manager Page on the admin side. Changes include:
- added API functions
- modified fields in officer schema
- modified routing and tests to reflect changes in schema
- added frontend to About Us Page and adding/editing/deleting ability to About Us Manager

Testing About Us Page:
1) On the general side, the page should look like this after the executive team and officers have been added.
![image](https://user-images.githubusercontent.com/54826539/105655575-216e6680-5e75-11eb-9a32-ae5070c340db.png)
![image](https://user-images.githubusercontent.com/54826539/105655619-3d720800-5e75-11eb-84e3-05a40fb362a4.png)

Features should include:
- on hovering over the picture, the executive cards should reveal descriptions and the officer circles should reveal the first name of the officer
![image](https://user-images.githubusercontent.com/54826539/105655889-def95980-5e75-11eb-9438-5cd66a0bb903.png)
![image](https://user-images.githubusercontent.com/54826539/105655939-ffc1af00-5e75-11eb-806c-f1b2df1460bc.png)
- on hovering, the executive cards should appear to pop-out of the page (can be seen in screenshot above)
- on clicking the linkedin icon on the bottom left corner of the executive cards, the linked in profile should open in a new tab.
- any additions/edits/deletions made on About Us Manager side should be reflected in About Us.  

Testing About Us Manager
2) Adding an Executive Team Member/Officer
- click the button labeled new officer + which will open a modal
- fill out the modal accordingly
3) Editing an Executive Team Member/Officer
- click on the executive cards to edit an Executive Team Member's profile
- click on the officer circles to edit an Officer's profile
4) Deleting an Executive Team Member/Officer
- click on the executive card/officer circle and click on the delete button
- confirmation modal should pop up and upon confirmation, the executive/officer will be deleted

Features:
- the form in the creating/editing modal should change depending on whether team selected in executive or officer
- a confirmation modal should pop-up on deletion
- if required fields are not filled, there should be a yellow colored alert at the bottom of the modal and the submit button should be disabled. 
- if required fields are filled, there should be a green colored alert and the submit button should be enabled.
